sdk/cosmosdb/cosmos/.prettierrc.json

```diff
@@ -1,4 +1,4 @@
 {
```

Comment:
You should be able to remove this file, and reference the shared one located at `sdk/.prettierrc.json` in the scripts instead.

Reply:
Probably better in a follow up PR together with changes from re-formatting, since there's some difference in the two configuration. Logged #3236.